modules/large-deployments/pages/hub-namespaces.adoc

```diff
@@ -1,23 +1,36 @@
 [[lsd-hub-namespaces]]
 = Hub Namespaces
 
-Hub supports 3 different namespaces.
+Hub XMLRPC API operates in a similar way to the {productname} API.
+For {productname} API documentation, see https://documentation.suse.com/suma.
 
-[[name-spaces]]
-[cols="1,1,1", options="header"]
-.Namespaces
-|===
-| Name      | Description  | Example
-| hub       | to target the Hub itself. | `client.hub.login(login, password)`
-| unicast   | to target a single Server registered in the Hub | `client.unicast.system.list_systems(hubKey, serverId)`
-| multicast | to target multiple Servers | `client.multicast.system.list_systems(hubKey, [serverId1, serverId2], [], [])`
-|===
+The Hub XMLRPC API exposes the same methods that are available from the server's XMLRPC APIs, with a few differences in parameter and return types.
+Additionally, the Hub XMLRPC API supports some Hub-specific end points which are not available in the {productname} API.
 
-* all XMLRPC API methods available in a single Server are exposed by the namespaces above.
-Generally speaking, they accept the same parameters and return the same values with the exceptions described below.
-* the `hubKey` can be obtained via 3 different login methods.
-For more detail, see authentication modes.
-* individual Server IDs can be obtained via `client.hub.listServerIds(hubKey)` (see examples section)
-* the `unicast` namespace assumes all methods receive hubKey and serverId as their first two parameters, then any other parameter as specified by the regular Server API.
-* the `multicast` namespace assumes all methods receive hubKey, a list of Server IDs, then lists of per-Server parameters as specified by the regular Server API.
-Return value will be an array, indexed per Server, of the results of individual Server calls.
\ No newline at end of file
+Hub supports three different namespaces:
+
+* The ``hub`` namespace is used to target the Hub Server.
```

Comment:
this is not entirely true. The "Hub" namespaces is used to target the Hub XMLRPC API server.
(there are not many methods actually under this namespace, maybe 3 or 4) 

For targetting the Hub directly you don't use any special namespace, you just use it as if it were a regular susemanager API 

EDIT: we can just leave it like this for Beta3 and fix it after if we are gonna release it